tests/metrics/test_pd_extractors.py

```diff
@@ -4,20 +4,94 @@
 import pandas as pd
 from sklearn.datasets import load_boston
 
+from fklearn.data.datasets import make_tutorial_data
 from fklearn.metrics.pd_extractors import (combined_evaluator_extractor,
                                            evaluator_extractor, extract,
                                            split_evaluator_extractor,
                                            temporal_split_evaluator_extractor)
 from fklearn.training.regression import linear_regression_learner
 from fklearn.validation.evaluators import (combined_evaluators, r2_evaluator,
                                            spearman_evaluator, split_evaluator,
-                                           temporal_split_evaluator)
+                                           temporal_split_evaluator, mse_evaluator)
 from fklearn.validation.splitters import (
     forward_stability_curve_time_splitter, out_of_time_and_space_splitter,
     stability_curve_time_splitter, time_learning_curve_splitter)
 from fklearn.validation.validator import validator
 
 
+def test__split_evaluator_extractor__when_split_value_is_missing():
```

Comment:
This test doesn't cover the case of a three depth tree, does it? Either way, using the `{}` guarantee any tree will work

Reply:
works for N